src/components/pages/Home.tsx

```diff
@@ -35,6 +35,18 @@ const queryLatest = {
   sort: { created: -1 }
 }
 
+const queryAlgorithms = {
+  page: 1,
+  offset: 9,
+  query: {
+    nativeSearch: 1,
+    query_string: {
+      query: `service.main.type == algorithm`
```

Comment:
this query seems to not work completely. Works fo me when connected to Rinkeby where algorithms are already there. But  when connected to mainnet, I get assets with `dataset` type shown in this section when it should just return nothing right now. 

And for this query we should exclude purgatory assets with `isInPurgatory` too, like we do for other queries https://github.com/oceanprotocol/market/pull/388/files#diff-1df2f2876c20c90c294da5abb9fcf0812eec3448de2aea5eb20e5e5f4a7d6cc7R20

Reply:
I understand, I'm working right now to correct this.